scipy/sparse/linalg/isolve/tests/test_minres.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.linalg import norm
+from numpy.testing import (assert_equal, assert_array_equal, assert_allclose,
+                           assert_, assert_raises, run_module_suite, dec)
+from scipy.sparse import diags, eye, lil_matrix
+from scipy.sparse.linalg.isolve import minres
+
+# A Random 10 x 10 sparse matrix
```

Comment:
Move this inside a function
```def get_matrix():```
Preferably no test code should run at import time.